application/src/main/java/uk/nhs/hee/tis/revalidation/validator/TraineeRecommendationRecordDTOValidator.java

```diff
@@ -45,32 +57,56 @@ public void validate(Object target, Errors errors) {
 
     if (errors.getErrorCount() == 0) {
       final var recordDTO = (TraineeRecommendationRecordDto) target;
+
       if (!StringUtils.hasLength(recordDTO.getGmcNumber())) {
         errors.reject("GmcNumber", "Gmc Number can't be empty or null");
+        return;
```

Comment:
This changes the logic where the GMC number is null & there was another error e.g. no `recommendationType`.  You could put the null check with the attempt to `validateIfDeferAllowed()`.